python/ray/util/client/__init__.py

```diff
@@ -1,8 +1,11 @@
-from typing import List, Tuple, Dict, Any
+from typing import List, Tuple, Dict, Any, TYPE_CHECKING
 from ray.job_config import JobConfig
 import os
 import sys
 import logging
+if TYPE_CHECKING:
+    from ray.util.client.common import ClientActorHandle
+    from ray.util.client.common import ClientObjectRef
```

Comment:
why do you need this?

Reply:
Is it okay to just import these?

Reply:
I think the import is necessary -- or at least my linter complains if I don't import.
As for the "if", I copy-pasted from here: https://github.com/ray-project/ray/blob/aa2f60b3d5ff3ac10278e0e007b19b6bee2ba066/python/ray/util/client/api.py#L6
Not sure why we need to check for `TYPE_CHECKING`, but might as well keep things consistent.